apis/management.cattle.io/v3/cluster_scan_types.go

```diff
@@ -78,3 +78,29 @@ type ClusterScan struct {
 	Spec   ClusterScanSpec   `json:"spec"`
 	Status ClusterScanStatus `yaml:"status" json:"status,omitempty"`
 }
+
+type CisBenchmarkVersionInfo struct {
+	MinKubernetesVersion string `yaml:"min_kubernetes_version" json:"minKubernetesVersion"`
+}
+
+type CisDefaultConfig struct {
+	BenchmarkVersion string `yaml:"benchmark_version" json:"benchmarkVersion"`
+}
+
+type CisConfig struct {
+	types.Namespaced
+
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	DefaultConfig CisDefaultConfig `yaml:"default_config" json:"defaultConfig,omitempty"`
```

Comment:
Confused as to why it's named `DefaultConfig`? It could either have BenchmarkVersion directly, or if empty would default to some value?

Reply:
At this point, we just have one field `BenchmarkVersion` but we could be adding more, hence created a struct.

Reply:
As requested renamed the struct